kompose.spec

```diff
@@ -235,20 +234,29 @@ done
 sort -u -o devel.file-list devel.file-list
 %endif
 
+# Why does check need to use buildroot at all? 
 %check
 %if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd):%{gopath}
+export GOPATH=$(pwd):%{gopath}
```

Comment:
hey @ingvagabund - I updated my spec file with this - see what you think:

```
# check uses %{buildroot} so that unit-tests can be run over the files
# that are about to be installed with the rpm. 
%check
%if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
%if ! 0%{?with_bundled}
export GOPATH=%{buildroot}/%{gopath}:%{gopath}
%else
# Since we aren't packaging up the vendor directory we need to link 
# back to it somehow. Hack it up so that we can add the vendor
# directory from BUILD dir as a gopath to be searched when executing
# tests from the BUILDROOT dir.
ln -s ./ ./vendor/src # ./vendor/src -> ./vendor
export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
%endif
```